src/debugger/rnDebugSession.ts

```diff
@@ -148,14 +148,15 @@ export class RNDebugSession extends DebugSessionBase {
         super.disconnectRequest(response, args, request);
     }
 
-    protected establishDebugSession(resolve?: (value?: void | PromiseLike<void> | undefined) => void): void {
+    protected establishDebugSession(attachArgs: IAttachRequestArgs, resolve?: (value?: void | PromiseLike<void> | undefined) => void): void {
```

Comment:
Do we take into account other possible arguments which may be contained in attachArgs? Like other js-debug arguments that could be important in such cases